locales/nb.schema.json

```
@@ -827,6 +845,17 @@
         },
         "social_sharing": {
           "name": "Knapper for deling på sosiale medier"
+        },
+        "share": {
+          "name": "Del",
+          "settings": {
+            "featured_image_info": {
+              "content": "Hvis du inkluderer en kobling i innlegg på sosiale medier, vil sidens fremhevede bilde vises som forhåndsvisningsbilde. [Finn ut mer] (https://help.shopify.com/en/manual/online-store/images/showing-social-media-thumbnail-images)."
```

This will create a broken link 
```suggestion
              "content": "Hvis du inkluderer en kobling i innlegg på sosiale medier, vil sidens fremhevede bilde vises som forhåndsvisningsbilde. [Finn ut mer](https://help.shopify.com/en/manual/online-store/images/showing-social-media-thumbnail-images)."
```